optimizely/project_config.py

```diff
@@ -557,3 +566,35 @@ def is_feature_experiment(self, experiment_id):
         """
 
         return experiment_id in self.experiment_feature_map
+
+    def get_variation_from_id_by_experiment_id(self, experiment_id, variation_id):
+        """ Gets experiment id and variation id
```

Comment:
Gets only variation by variation id under specific experiment id.

Reply:
please rephrase

Reply:
comments changed to better describe what is happening.